webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -244,8 +244,8 @@ def bugzilla_submit_url(report, bug_product):
         'product': bug_product,
         'op_sys': op_sys,
         'rep_platform': report['cpu_name'],
-        'cf_crash_signature': '[@ {}]'.format(report['signature']),
-        'short_desc': 'Crash in {}'.format(report['signature']),
+        'cf_crash_signature': u'[@ {}]'.format(report['signature']),
+        'short_desc': u'Crash in {}'.format(report['signature']),
```

Comment:
I'd force `report['signature']` to be a string. That allows us to keep the machinery between Socorro and Bugzilla simple (it always uses strings) which is easier to test and less likely to break if anything in Bugzilla changed.